tests/testUtil.ts

```diff
@@ -1,6 +1,7 @@
-import { APIGatewayProxyEvent, Context } from "aws-lambda"
+/* eslint-disable @typescript-eslint/ban-types */
+import { APIGatewayProxyEvent, Context, SQSEvent } from "aws-lambda"
 
-export const createAPIGatewayEvent = ({
+export const createApiGatewayEvent = ({
```

Comment:
`createAPIGatewayEvent` seems more consistent with the rest of the project (uppercase abbreviations)